Mathlib/LinearAlgebra/SesquilinearForm.lean

```diff
@@ -304,6 +304,12 @@ theorem ortho_comm {x y} : IsOrtho B x y ↔ IsOrtho B y x :=
   H.isRefl.ortho_comm
 #align linear_map.is_alt.ortho_comm LinearMap.IsAlt.ortho_comm
 
+theorem eq_of_add_add_eq_zero {a b c : M₁} (hAdd : a + b + c = 0) : B a b = B b c := by
```

Comment:
You should move this up to the `AddCommMonoid` section above, where the `IsCancelAdd` argument will need to return.

You don't need it here because you have `AddCommGroup`.

Reply:
So if I understood correctly, since the theorem itself is true for `AddCommMonoid`, not just for `AddCommGroup`, it is better to be in `AddCommMonoid` section with `IsCancelAdd` argument, right?